clinica/converters/oasis_to_bids/_converter.py

```diff
@@ -170,13 +172,10 @@ def convert_single_subject(subj_folder: Path, dest_dir: Path):
         session_folder = bids_subj_folder / "ses-M000"
         if not session_folder.is_dir():
             (session_folder / "anat").mkdir(parents=True, exist_ok=True)
+
         # In order do convert the Analyze format to Nifti the path to the .img file is required
-        try:
-            img_file_path = next(t1_folder.glob("*.img"))
-        except StopIteration:
-            msg = f"No file ending in .img found in {t1_folder}."
-            cprint(msg, lvl="error")
-            raise FileNotFoundError(msg)
+        img_file_path = get_first_image(t1_folder)
```

Comment:
Since we are only using this variable in the next line, I think we can just get rid of it:

```python
nb.save(
    _get_image_with_good_orientation(
        get_first_image(t1_folder),
        session_folder / "anat" / f"{participant_id}_ses-M000_T1w.nii.gz",
    )
)
```